### PR TITLE
refactor!: replace `compute_commitment_for_testing` with `RistrettoPoint::compute_commitments`

### LINKED ISSUE #168: test: replace `compute_commitment_for_testing` with `RistrettoPoint::compute_commitments`
# Background and Motivation
The `compute_commitment_for_testing` function was a utility function to simplify computing commitments for testing purposes. Newer interfaces simplify this code, so this function is no longer needed, and adds clutter.

# Requested Changes
- [ ] Replace `compute_commitment_for_testing` with `RistrettoPoint::compute_commitments`, which exists because of an `impl Commitment for RistrettoPoint`.

Note:
`compute_commitment_for_testing` could be refactored to leverage `Commitment::compute_commitments` like this:
```rust
let vals = iter_cast::<Curve25519Scalar, [u64; 4]>(slice_cast_to_iter(vals));
let mut commitments = [RistrettoPoint::default()];
Commitment::compute_commitments(
    &mut commitments,
    &[CommittableColumn::Scalar(vals)],
    offset_generators,
    &(),
);
commitments[0]
```

### PR DESCRIPTION
**Pull request by Gmin2**
# Rationale for this change

Mentioned in #168 

# What changes are included in this PR?

Replaced the `compute_commitment_for_testing` to `RistrettoPoint::compute_commitments`,in the test case that was being used 

Fixes #168 
/claim #168

### CODE REVIEW

File: crates/proof-of-sql/src/sql/proof/verifiable_query_result_test_utility.rs [hunk 1]
```diff
@@ -4,16 +4,17 @@ use super::{
...
+        scalar::Curve25519Scalar,
+        slice_ops::slice_cast,
     },
     sql::proof::Indexes,
 };
 use blitzar::proof::InnerProductProof;
 use curve25519_dalek::{ristretto::RistrettoPoint, traits::Identity};
 use num_traits::One;
 use serde::Serialize;
-
```
Comment (JayWhite2357) [inline crates/proof-of-sql/src/sql/proof/verifiable_query_result_test_utility.rs:16]: NIT: please remove this superfluous deletion

File: crates/proof-of-sql/src/base/database/owned_table_test_accessor_test.rs [hunk 2]
```diff
@@ -130,7 +133,16 @@ fn we_can_access_the_commitments_of_table_columns() {
...
     assert_eq!(
         accessor.get_commitment(column),
-        compute_commitment_for_testing(&[4, 5, 6], 0_usize)
+        RistrettoPoint::compute_commitments(
+            &[CommittableColumn::Scalar(slice_cast::<
+                Curve25519Scalar,
+                [u64; 4],
+            >(&slice_cast(&[
+                4, 5, 6
+            ])))],
```
Comment (JayWhite2357) [inline crates/proof-of-sql/src/base/database/owned_table_test_accessor_test.rs:142]: You should be able to leverage the conversions we have in place for `CommittableColumn` to make it much more readable. See here: https://github.com/spaceandtimelabs/sxt-proof-of-sql/blob/e40362a2865fb9c95ea5bdfb9bbc48471500cb33/crates/proof-of-sql/src/base/commitment/committable_column.rs#L152-L188.

Something like this should work
```suggestion
            &[(&[4, 5, 6]).into()],
```

**Review by JayWhite2357 — CHANGES_REQUESTED** [review]
Thanks for the PR.
Please remove the now-unused `compute_commitment_for_testing` method.

Also, be sure to run the tests you are modifying to make sure they are correct: one of them is failing in the CI.

**Comment by Gmin2** [comment]
Done with the changes @JayWhite2357

**Review by JayWhite2357 — APPROVED** [review]
Thanks!

**Comment by JayWhite2357** [comment]
/tip $10
Sorry for the confusion on the bounty amounts.

**Pull request merged**